lib/vrl/stdlib/src/encode_logfmt.rs

```diff
@@ -225,9 +225,10 @@ mod tests {
             args: func_args![value:
                 btreemap! {
                     "lvl" => "info",
-                    "msg" => r#"payload: {"code": 200}\n"#
+                    "msg" => r#"payload: {"code": 200}\n"#,
+                    "another_field" => "some\nfield\\and things"
```

Comment:
What happens if the key has a space in it? This would be possible with Vrl..

Reply:
It is quoted in the same way that values are quoted. I added another field to test that in 5716e7820

Reply:
Perfect! Yeah this is looking good.